app/Member.php

```diff
@@ -22,10 +22,20 @@ class Member extends Model
      *
      * @var array
      */
-    protected $fillable = array(
```

Comment:
That's quite true! Good point. Maybe in that case, it would be good to use a whitelist (`$fillable`) and remove the blacklist (`$guarded`).